### ISSUE TITLE
Stale documentation for fasparquet.write

### ISSUE THREAD

**Issue by eadanfahey** [dask/fastparquet#486]
The documentation for the `compression` argument to [`fastparquet.write`](https://fastparquet.readthedocs.io/en/latest/api.html#fastparquet.write) is out of date, and the provided example produces the error:

```
File "~/.local/lib/python3.6/site-packages/fastparquet/compression.py", line 69, in lz4_compress
    return lz4.block.compress(data, **kwargs)
TypeError: 'compression_level' is an invalid keyword argument for this function
```

The error is because the API to `python-lz4` library [has changed](https://python-lz4.readthedocs.io/en/stable/lz4.block.html#lz4.block.compress) (`compression_level` and `content_checksum` are no longer arguments of `lz4.block.compress`). A more up-to-date example could be:

```
{
    col1: {
        "type": "LZ4",
        "args": {
            "mode": "high_compression",
            "compression": 9
         }
    },
    col2: {
        "type": "SNAPPY",
        "args": None
    }
    "_default": {
        "type": "GZIP",
        "args": None
    }
}
```

**Comment by martindurant** [comment]
Thanks for pointing that out. Would you be willing to provide a PR with a change reflecting the new reality (and perhaps mention the version of lz4 it requires)?

**Comment by eadanfahey** [comment]
Sure thing. I made a pull request updating the docs to match my original post. Regarding the version, the current doc example doesn't work with any version of `python-lz4` (latest version is `3.0.2` and I tested back to `0.10`).  Perhaps, it's because the current example seems to using the arguments of [`lz4.frame.compress`](https://python-lz4.readthedocs.io/en/stable/lz4.frame.html#lz4.frame.compress) rather than those of [`lz4.block.compress`](https://python-lz4.readthedocs.io/en/stable/lz4.block.html#lz4.block.compress).
